### DISCUSSION TITLE
Recommended way for setting service user (Linux)?

### DISCUSSION THREAD

**Discussion by HubKing** [TriliumNext/Trilium#2845] [Q&A] [answered]
For testing, I used my own user account, which has `sudo`. It is working, but probably this is not a good for security reasons. What is the recommended way to use a different account? I guess I would create a new account, but, of course, I will not log in with that account. This page ( https://superuser.com/questions/77617/how-can-i-create-a-non-login-user )  says `useradd -M`, but Trilium needs the home directory, so that won't do.

```
[Service]
User=xxx
Group=xxx
Type=simple
ExecStart=/opt/trilium/trilium.sh
WorkingDirectory=/opt/trilium/
```

**Comment by zadam** [comment, marked as answer]
Trilium doesn't need a home directory. You can use `TRILIUM_DATA_DIR` env. variable to point it to any other directory (to which the user has permissions).
